### PR TITLE
More whatsnew entries for 5.1

### PR DESCRIPTION
**Pull request by takluyver**
I've tried to stick to user-visible changes, so I haven't listed every PR.

### CODE REVIEW

File: docs/source/whatsnew/version5.rst [hunk 1]
```diff
@@ -13,6 +13,18 @@ IPython 5.1
 * Deprecate ``IPtyhon.core.debugger.Tracer()`` in favor of simpler, newer, APIs. :ghpull:`9731`
```
Comment (willingc) [inline docs/source/whatsnew/version5.rst]: Typo in IPython

**Comment by willingc** [comment]
@takluyver minor typo in an existing line. Overall, looks good to me.

File: docs/source/whatsnew/version5.rst (hunk 1 above)
Comment (takluyver) [inline docs/source/whatsnew/version5.rst]: Thanks, fixed

**Comment by willingc** [comment]
Thanks @takluyver

**Pull request merged**